ioctl/cmd/ws/wsrouterbinddapp.go

```diff
@@ -59,10 +61,16 @@ func init() {
 }
 
 func bindDapp(projectID uint64, dapp string) (string, error) {
-	addr, err := address.FromString(dapp)
+	addrStr, err := util.Address(dapp)
 	if err != nil {
 		return "", errors.Wrapf(err, "invalid dapp address: %s", dapp)
 	}
+	// Convert addrStr to an Address type
```

Comment:
nit: can remove this since the func name clearly shows it, but adding comment is a good habit!